02-wrangling.Rmd

```diff
@@ -328,35 +519,39 @@ through and find where `output_1` and `output_2` are used in each subsequent lin
 
 Another option for doing this would be to *compose* the functions:
 
-```
+```{r eval = F}
 output <- select(filter(mutate(data, new_col = old_col * 2), other_col > 5), new_col)
 ```
 
 Code like this can also be difficult to understand. Functions compose (reading from left to right) in the *opposite order* in which
 they are computed by R (above, `mutate` happens first, then `filter`, then `select`). It is also just a really long line of code
 to read in one go. 
 
-The *pipe operator* `%>%` solves this problem, resulting in cleaner and easier-to-follow code. The below accomplishes the same thing as the previous two code blocks:
+The *pipe operator* `|>` solves this problem, resulting in cleaner and easier-to-follow code. The below accomplishes the same thing as the previous two code blocks:
 
-```
-output <- data %>% 
-            mutate(new_col = old_col * 2) %>%
-            filter(other_col > 5) %>%
+```{r eval = F}
+output <- data |> 
+            mutate(new_col = old_col * 2) |>
+            filter(other_col > 5) |>
             select(new_col)
 ```
 
+> Note `%>%` is the previous version of the pipe operator so you may see it used 
+in some textbooks or other sources. There are some drawbacks to using the `%>%` 
+pipe operator, thus, in this textbook we will be using the new pipe operator syntax `|>`. 
```

Comment:
We should expand this a bit. We should add that `%>%` is not built in to R, and needs to be loaded via an external R package. Also that `|>` was inspired by `%>%` (interesting history). And note any key differences that might be of interest to our learners. If the differences seem too complex, say this is so and point them to where they can learn more. Wonder if we want to demonstrate one use of it, showing they do the same thing?